src/layout/header/header.js

```diff
@@ -96,6 +98,9 @@ const Header = ({
         <Typography variant="body2" color="primary" component={NavLink} to={'/faq'} activeClassName={classes.activeLink}>
             FAQ
         </Typography>
+        <Typography variant="body2" color="primary" component={NavLink} to={'/blog'} onClick={refreshPage} activeClassName={classes.activeLink}>
```

Comment:
We should not need to refresh the page to get the blog to work. This is a violation of what a single page application is. Please find another way to handle this. I recommend you move the initialization code into the Blog component and dynamically load it with javascript in a useEffect hook